cmd/assetsvc/handler_test.go

```diff
@@ -179,10 +196,10 @@ func Test_newChartListResponse(t *testing.T) {
 			clResponse := newChartListResponse(tt.input)
 			assert.Equal(t, len(clResponse), len(tt.result), "number of charts in response should be the same")
 			for i := range tt.result {
-				assert.Equal(t, clResponse[i].Type, "chart", "response type is chart")
-				assert.Equal(t, clResponse[i].ID, tt.result[i].ID, "chart ID should be the same")
-				assert.Equal(t, clResponse[i].Relationships["latestChartVersion"].Data.(models.ChartVersion).Version, tt.result[i].ChartVersions[0].Version, "latestChartVersion should match version at index 0")
-				assert.Equal(t, clResponse[i].Links.(selfLink).Self, pathPrefix+"/ns/"+namespace+"/charts/"+tt.result[i].ID, "self link should be the same")
+				assert.Equal(t, "chart", clResponse[i].Type, "response type is chart")
+				assert.Equal(t, getChartID(tt.result[i].Repo.Name, tt.result[i].Name), getChartID(tt.result[i].Repo.Name, tt.result[i].Name), "chart ID should be the same")
```

Comment:
Good catch, thx! It's my bad; I updated many asserts because they were flipped `(actual, expected)` instead of `(expected, actual)` and I just did a wrong copy&pase...